[Bug] make debug_with_test fails at ParquetReaderTest.cpp:88:1: error: control reaches end of non-void function

### Component Selection

- [x] Core Engine (Expression eval, Memory, Vector)
- [ ] Connectors / File Formats (Hive, Parquet, etc.)
- [ ] API / Bindings (Python, etc.)
- [ ] Build
- [ ] Other

### Describe the Bug

Run command `make debug_with_test` gets the following error.
```
/home/zhongjun.jin/bolt/bolt/dwio/parquet/tests/reader/ParquetReaderTest.cpp: In function ‘std::string {anonymous}::getVariantFixturePath(const std::string&)’:
/home/zhongjun.jin/bolt/bolt/dwio/parquet/tests/reader/ParquetReaderTest.cpp:88:1: error: control reaches end of non-void function [-Werror=return-type]
   88 | }
      | ^
```

### Reproduction Steps

Run command `make debug_with_test`

### Bolt Version / Commit ID

9573968e03216e507f5f17e05c13cc376a66eb7f

### System Configuration

```markdown
- **OS**: (e.g. Ubuntu 22.04, CentOS 7)
- **Compiler**: (e.g. GCC 11, Clang 14)
- **Build Type**: (Debug / Release / RelWithDebInfo)
- **CPU Arch**: (e.g. x86_64 AVX2, ARM64)
- **Framework**: (e.g. Spark 3.3, PrestoDB)
```

### Logs / Stack Trace

```shell

```

### Expected Behavior

_No response_

### Additional context

_No response_

--- a/bolt/dwio/parquet/tests/reader/ParquetReaderTest.cpp
+++ b/bolt/dwio/parquet/tests/reader/ParquetReaderTest.cpp
@@ -79,8 +79,7 @@ std::string getVariantFixturePath(const std::string& fileName) {
     }
   }
 
-  BOLT_CHECK(
-      false,
+  BOLT_FAIL(
       "VARIANT fixture not found: {}, cwd: {}, source: {}",
       fileName,
       cwd.string(),

